rootfs/etc/nginx/lua/balancer/ewma.lua

```diff
@@ -170,20 +177,50 @@ function _M.balance(self)
   if #peers > 1 then
     local k = (#peers < PICK_SET_SIZE) and #peers or PICK_SET_SIZE
     local peer_copy = util.deepcopy(peers)
```

Comment:
I think this is unnecessary now, no? Because below you effectively do the same by copying `peers` to `filtered_peers`?